src/commands/connections.ts

```diff
@@ -191,41 +191,7 @@ export async function deleteDirectConnection(item: DirectEnvironment) {
   await DirectConnectionManager.getInstance().deleteConnection(item.connectionId);
 }
 
-// XXX: the UI for this was replaced by editDirectConnection. Keeping in case we want to expose it again in the future elsewhere.
-export async function renameDirectConnection(item: DirectEnvironment) {
```

Comment:
It's been a while since we disabled this, and as far as I know we don't have any intention to revert in the near term; might as well get rid of it and improve our test coverage!